R/statistic-histogram.R

```diff
@@ -2,6 +2,32 @@
 #' 
 #' Determine accuracy of histogram release, given epsilon and delta, for the differentially 
 #' private histogram release.
+#' 
+#' In differential privacy, "accuracy" is defined as the threshold value above which a given value 
+#' is "significantly different" from the expected value. Mathematically, this is written as:
+#' \deqn{\alpha = Pr[Y > a]}
+#' Where \eqn{\alpha} is the statistical significance level, \eqn{a} is the accuracy, and \eqn{Y} is a random 
+#' variable following the Laplace distribution.
+#' 
+#' In other words, the accuracy is a threshold value, above which a value is "unexpected". We call
+#' this "accuracy" because most values will be below this threshold, and thus are "accurate" enough
+#' to be considered "expected" values.
+#' 
+#' Deriving the accuracy formula:
+#' 
+#' \enumerate{
+#'     \item The probability density function (PDF) \eqn{f(x)} of the Laplace distribution is:\cr
+#'           \eqn{f(x) = {1 / 2\lambda} * e^{-|x-\mu| / \lambda}}
+#'     \item For differential privacy calculations \eqn{\mu = 0}, because the "average" amount of noise added is \eqn{0} noise
+#'     \item For differential privacy, "accuracy" is defined as the magnitude of difference between the expected and observed values, so the distrubution is "folded", or doubled, because we are only concerned with the absolute value
```

Comment:
I'd try to shorten this a little. We already know that it's folded from the definition above, so shouldn't restate it here (unless very concisely).